main.cpp

```diff
@@ -403,6 +406,8 @@ Config parse_commandline_opts(int argc, char **argv) {
                 if (i < argc) {
                     config.tmx_filename = argv[i];
                 }
+            } else if (strcmp(cmd, "binary") == 0) {
+                cfg_output_bin = true;
```

Comment:
I went ahead and made the changes. Didn't add the config parameter to the sms_cl123 function since it isn't used. Sacrifices a bit of consistency in sake of compiler warnings. Let me know what you think. Thanks.